config/defaults.ini

```diff
@@ -202,13 +202,8 @@ kalite_content_user = uploader
 kalite_content_variant = endless
 kalite_content_version = 0.16
 
-# Preloaded Kolibri content. eos-kolibri-help is always selected by default.
+# Set to true to enable preloaded Kolibri content
 kolibri_content = false
```

Comment:
Perhaps we should get rid of this `kolibri_content` variable too, and check whether `EIB_KOLIBRI_INSTALL_CHANNELS` is empty or not to decide if we should bail out early in the hooks.

Reply:
Yeah, makes sense. Done :)